app/code/Magento/Checkout/Block/Cart.php

```diff
@@ -245,4 +250,22 @@ public function getPagerHtml()
     {
         return $this->getChildHtml('pager');
     }
+
+    /**
+     * Checks is clear cart action enabled
+     *
+     * @codeCoverageIgnore
+     * @return boolean
+     */
+    public function isClearCartEnabled()
+    {
+        $isEnabled = $this->_getData('clear_cart_enabled');
+        if ($isEnabled === null) {
+            $isEnabled = $this->_scopeConfig->getValue(
+                self::XML_CONFIG_CLEAR_CART_ENABLED,
+                \Magento\Store\Model\ScopeInterface::SCOPE_STORE
```

Comment:
Can you please move this into "use" section and use here relative path?

Reply:
@nuzil 
> Can you please also cover changes with MFTF tests?

Added MFTF in the checkout module.